astropy/tests/tests/test_imports.py

```diff
@@ -43,12 +32,9 @@ def test_imports():
         raise AttributeError('package to generate config items for does not '
                              'have __file__ or __path__')
 
-    excludes = _py2_packages
-
     prefix = package.__name__ + '.'
 
     def onerror(name):
-        if not any(name.startswith(excl) for excl in excludes):
             # A legitimate error occurred in a module that wasn't excluded
```

Comment:
Could be unindented.

Reply:
well, ``excludes`` doesn't exist any more. (and unless I mess up something horribly, this isn't working for a long time now, as neither ``astropy.extern.configobj_py2`` nor ``astropy.extern.configobj_py3`` exist any more).

Reply:
hm, I meant that it doesn't need 8 space indentation now that the `if` is removed. Am I wrong?

Reply:
Oops, it seems that I just can't read. unintended vs unindented